docs/migration-guide.md

```diff
@@ -27,14 +27,17 @@ The type `sds` is removed from the public API.
 
 ## Migrating from `hiredis-cluster` 0.14.0
 
-The command used to update the internal slot map is changed to `CLUSTER SLOTS`.
-`CLUSTER NODES` can be re-enabled through options, see `VALKEY_OPT_USE_CLUSTER_NODES`.
+* The cluster client initiation procedure is changed and `valkeyClusterOptions`
+  should be used to specify options when creating a context.
+  The `examples` directory in this repository contains some common client
+  initiation examples that might be helpful.
```

Comment:
We can also link to cluster.md from this migration guide, right? I think cluster.md should have some example of async connect with options.

Reply:
Added links in the migration guide to the newly added examples in cluster.md